libs/platform/src/lib/components/form/combo-box/combo-box.component.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, ViewChild, ElementRef } from '@angular/core';
+import { ComboBoxComponent } from '@fundamental-ngx/platform';
+import { ComponentFixture, TestBed, async } from '@angular/core/testing';
+
+
+@Component({
+    selector: 'fdp-test-combo-box',
+    template: `
+        <fdp-combo-box [dataSource]="addressDataSource" displayKey="toString" placeholder="select from here"></fdp-combo-box>
+    `
+})
+class TestWrapperComboBoxComponent {
+
+    @ViewChild(ComboBoxComponent, { static: true })
+    selectRef: ComboBoxComponent;
+
+    @ViewChild(ComboBoxComponent, { read: ElementRef, static: true })
+    selectElement: ElementRef;
+
+    wrapperValue: string;
+}
+describe('ComboBoxComponent', () => {
+    let component: ComboBoxComponent;
+    let fixture: ComponentFixture<ComboBoxComponent>;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [ComboBoxComponent]
+        })
+            .compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(ComboBoxComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
```

Comment:
We should add some real unit tests here. This test is the default test and does not add to our code coverage.

Adds some test which include DataProvider and DataService and which tests that when you type a keyword in the combobox you get matching results in the combobox dropdown.

Reply:
It looks like the unit tests still need some attention.